apps/gitness/src/components-v2/toaster.tsx

```diff
@@ -1,23 +1,21 @@
-import { Toast, ToastClose, ToastDescription, ToastTitle, ToastViewport, useToast } from '@harnessio/ui/components'
+import { Toast, useToast } from '@harnessio/ui/components'
 
 export function Toaster() {
   const { toasts } = useToast()
 
   return (
     <>
-      {toasts.map(function ({ id, title, description, action, ...props }) {
-        return (
-          <Toast key={id} {...props}>
-            <div className="grid gap-1">
-              {title && <ToastTitle>{title}</ToastTitle>}
-              {description && <ToastDescription>{description}</ToastDescription>}
-            </div>
-            {action}
-            <ToastClose />
-          </Toast>
-        )
-      })}
-      <ToastViewport />
+      {toasts.map(({ id, title, description, action, ...props }) => (
+        <Toast.Root key={id} {...props}>
+          <div className="grid gap-1">
+            {title && <Toast.Title>{title}</Toast.Title>}
+            {description && <Toast.Description>{description}</Toast.Description>}
```

Comment:
```suggestion
            {!!title && <Toast.Title>{title}</Toast.Title>}
            {!!description && <Toast.Description>{description}</Toast.Description>}
```
Both the title and description are ReactNodes, so value `0` will render `0`

Reply:
I was just name-spacing here, but it's a good point.